src/components/account-created/index.njk

```diff
@@ -28,4 +28,6 @@
     }) }}
 {% endif %}
 </form>
+
+{{ga4OnPageLoad({ nonce: scriptNonce,statusCode:"200",englishPageTitle:pageTitleName, taxonomyLevel1:"authentication", taxonomyLevel2:"create account", contentId:"6857e410-75b8-4807-b475-3f24fc96c9de",loggedInStatus:true,dynamic:true})}}
```

Comment:
Global issue: Dynamic needs to be "true" only if the nunjuck file is used in different scenarios / user journeys.
Have you check that before to set "true"?